rust/rebase/src/subject/ethereum.rs

```diff
@@ -31,6 +31,12 @@ impl Subject for Eip155 {
         Ok(self.address.clone())
     }
 
+    fn verification_method(&self) -> Result<String, SubjectError> {
+        Err(SubjectError::Validation(
+            "Ethereum doesn't currently support verification method!".to_string(),
```

Comment:
do we use `did:pkh:eip155` for ethereum here? if so we can support it with `did:pkh:eip155:X:<account>#blockchainAddress`

Reply:
That's really good to know, I'll change it to take that approach.